src/pages/stacks-blockchain/transactions.md

```diff
@@ -0,0 +1,487 @@
+---
+title: Transactions
+description: Guide to Stacks 2.0 transactions
+---
+
+## Introduction
+
+Transactions are the fundamental unit of execution in the Stacks blockchain. Each transaction is originated from a standard account, and is retained in the Stacks blockchain history for eternity. This guide helps you understand Stacks 2.0 transactions.
+
+If you want to jump right in and broadcast your first transaction, try this tutorial:
+
+[@page-reference | inline]
+| /stacks-blockchain/sending-tokens
+
+-> The information on this page is based on a design proposal. You can find more conceptual details in this document: [SIP 005: Blocks, Transaction, Accounts](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md).
+
+## Lifecycle
+
+Transactions go through phases before being finally confirmed, and available for all, on the Stacks 2.0 network.
+
+![Transaction lifecycle](/images/tx-lifecycle.png)
+
+- **Generate**: Transactions are assembled according to the encoding specification.
+- **Verify and sign**: Transactions are verified to confirm if they are well-formed. Required signatures are filled in.
```

Comment:
While the transaction libs do try to ensure the transaction is valid. The verification step happens on the node once it's broadcasted.

Reply:
thanks!